docs/_data/components/schemas/limit.yaml

```diff
@@ -30,10 +33,12 @@ oneOf:
         format: float
         minimum: 1
         maximum: 10000
+    required:
+      - mva
   - type: object
     title: Current
     description: >-
-      Representation of a limit in amps.  Assumes that TROLIE has a nominal voltage
+      Representation of a limit in amps. Assumes that TROLIE has a nominal voltage
```

Comment:
This is an implementation detail left to the vendor. I believe the intention with the comment is to acknowledge that you cannot control to amperes without knowing the voltage and further assumes that a nominal/design voltage is acceptable in any case for TROLIE. This may not make sense in an implementation that makes use of a voltage solution. @getorymckeag do you have any thoughts here?